napari/_vendor/qt_json_builder/qt_jsonschema_form/form.py

```diff
@@ -28,6 +31,7 @@ class WidgetBuilder:
             "plugins": widgets.PluginWidget,
             "shortcuts": widgets.ShortcutsWidget,
             "extension2reader": widgets.Extension2ReaderWidget,
+            "dask": widgets.DaskSettingsWidget,
```

Comment:
Where the name `dask` is set? It is a field name?

Reply:
In `preferences_dialog.py`, there is a reference dictionary where you specify certain widgets to be used when the fields from certain settings are to be built.
ui_schema = {
        "call_order": {"ui:widget": "plugins"},
        "highlight_thickness": {"ui:widget": "highlight"},
        "shortcuts": {"ui:widget": "shortcuts"},
        "extension2reader": {"ui:widget": "extension2reader"},
        "dask": {"ui:widget": "dask"},
    }